taranveersingh93/rancid-tomatillos
# Feature/style homepage

#### What is the feature?
- Finesse Styling of various elements on the homepage. 

#### How you implemented the solution?
- Added a neon logo animation using keyframes and making 3 different classes of the same animation taking different times, attaching them to different elements in the logo. 
- Add conditional rendering of homepage
- Make grid responsive
- Add a key for each card
- Add movie ID for each card for later use
- Add hover animations on cards
- Add scroll animation on cards

#### Any changes in the UI (Screenshots)?
-
![Screenshot 2023-07-07 at 12 03 19 AM](https://github.com/taranveersingh93/rancid-tomatillos/assets/122247155/e57b1d54-ac16-4c20-956c-54ef01698f70)
![Screenshot 2023-07-07 at 12 03 03 AM](https://github.com/taranveersingh93/rancid-tomatillos/assets/122247155/d91e3ac5-5da9-45e2-9563-3f8609011d02)

#### How to test the feature (A test scenario or any new setup)?
- Ensure that landing page works
- Ensure no errors in console
- Resize the window to see the consistent sizing
- Hover over the cards to see cursor change to pointer and image size increasing

#### Any other notes:
- I used an npm package. Please run the following command in your terminal.
`npm install --save aos@next`
